packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -135,19 +131,19 @@ library TaikoData {
         // Slot 7: never or rarely changed
         uint64 genesisHeight;
         uint64 genesisTimestamp;
-        uint16 adjustmentQuotient;
+        uint16 __reserved70;
```

Comment:
Reviewers, please make sure you check the data layout shall be compatible with alpha-3